src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs

```diff
@@ -112,5 +144,28 @@ private static int GetInstanceFeatureScore(MSBuildInstance i)
 
             return score;
         }
+
+        public static Version GetSdkMinimumMSBuildVersion(DotNetInfo dotNetInfo, ILogger logger)
+        {
+            if (dotNetInfo is null
+                || string.IsNullOrWhiteSpace(dotNetInfo.SdksPath)
+                || dotNetInfo.SdkVersion is null)
+            {
+                return s_minimumMSBuildVersion;
+            }
+
+            var version = dotNetInfo.SdkVersion;
+            var sdksPath = dotNetInfo.SdksPath;
+            var minimumVersionPath = Path.Combine(sdksPath, version.ToNormalizedString(), "minimumMSBuildVersion");
```

Comment:
this is really there?! that's new to me!

Reply:
I was happy to find it =)